tests/phpunit/CRM/Contribute/Form/ContributionTest.php

```diff
@@ -859,19 +859,26 @@ public function testSubmitWithOutSaleTax() {
   public function testReSubmitSaleTax() {
     $this->enableTaxAndInvoicing();
     $this->relationForFinancialTypeWithFinancialAccount($this->_financialTypeId);
-    $form = new CRM_Contribute_Form_Contribution();
+    list($form, $contribution) = $this->doInitialSubmit();
+    $this->assertEquals(110, $contribution['total_amount']);
+    $this->assertEquals(10, $contribution['tax_amount']);
+    $this->assertEquals(110, $contribution['net_amount']);
```

Comment:
@eileenmcnaughton you already checking this values in doInitialSubmit() function (https://github.com/civicrm/civicrm-core/pull/9984/files#diff-f5dec9266a02875f07e5948eed1ca68bR1009). Any reason why you re-checking them again?